source/common/stream_info/utility.cc

```diff
@@ -116,6 +116,17 @@ absl::optional<std::chrono::nanoseconds> TimingUtility::lastDownstreamRxByteRece
   return duration(timing.value().get().lastDownstreamRxByteReceived(), stream_info_);
 }
 
+absl::optional<std::chrono::duration<double, std::milli>>
+TimingUtility::recordDuration(absl::optional<MonotonicTime> start_time) {
+  if (start_time.has_value()) {
+
+    auto ext_authz_duration =
```

Comment:
Nit. Probably choose a different variable name? Since this is not a specific file for ext_authz. Or move it inside the filter code since that is the only one using this?

Reply:
Changed the variable name.